Casks/mark-text.rb

```diff
@@ -15,4 +18,14 @@
     "~/Library/Saved Application State/com.github.marktext.marktext.savedState",
     "~/Library/Logs/marktext",
   ]
+
+  caveats do
+    "The apple silicon (arm) version of #{token} is not signed, and may
+display an error stating it is damaged and can't be openedif it is not
+installed installed using the --no-quarantine option.
+
+You can also use the command:
+    xattr -dr com.apple.quarantine /Applications/MarkText.app
+to allow it to run if installed without the --no-quarantine option."
```

Comment:
We don't want to recommend users disable macOS security features.

```suggestion
    "The apple silicon (arm) version of #{token} is not signed, and will
display an error stating it is damaged and can't be opened.
```